docs/request-send-flow.md

```diff
@@ -1,4 +1,4 @@
 Postman Runtime handles all the steps required to send a request. This flowchart
 provides an overview of the steps:
 
-![Request Flow](request-flow.png)
+![Request Flow](./img/request-flow.png)
```

Comment:
```suggestion
![Request Flow](/img/request-flow.png)
```

Reply:
Hi @codenirvana , 
Tried your suggested path `/img/request-flow.png` , it fails because it it treated as absolute path (i.e., root / img / request-flow.png) . The fix to problem is to slightly modify the `staticFiles` path to `./docs` . Which now renders `request-flow.png` at both locations.

Reply:
I have tested it , with above mentioned changes and it works perfectly fine. Updating the `.jsdoc-config.json`